### ISSUE TITLE
E721 false positive

### ISSUE THREAD

**Issue by eric-bonfadini** [PyCQA/pycodestyle#849]
With the following code I get a false positive for "E721 do not compare types, use 'isinstance()'".
From my tests, this happens if the function name ends with istype and it has at least two arguments.

```python
def func_histype(arg1, arg2, arg3):
    pass

```

**Comment by sigmavirus24** [comment]
What version are you using? Can you give us any kind of diagnostic information?

**Comment by eric-bonfadini** [comment]
```bash
pycodestyle --version
2.5.0
```

**Comment by FichteFoll** [comment]
I can confirm this on 2.5.0.

**Comment by asottile** [comment]
try out the branch in #850 :tada:

**Comment by eric-bonfadini** [comment]
I tried out that branch, and my use case works fine, thanx!
